src/pages/charter.js

```diff
@@ -243,12 +243,27 @@ const CharterPage = ({ location: {pathname} }) => {
       </ol>
       <h2>8) Trademarks</h2>
       <ol>
-        <li>Any trademarks relating to the Directed Fund or the Technical Projects<a href="#clarification-1"><sup>1</sup></a> , including
+        <li>Any trademarks relating to the Directed Fund or the Technical Projects, including
         without limitation any mark relating to any conformance program, must be transferred to
         and held by LF Projects, LLC or the Linux Foundation and available for use pursuant to
         LF Projects, LLC’s trademark usage policy, available at <a href="https://www.lfprojects.org/trademarks/" target="_blank" rel="noopener noreferrer">https://www.lfprojects.org/trademarks/</a>.</li>
       </ol>
-      <h2>9) Antitrust Guidelines</h2>
+      <h2>9) Technical Projects</h2>
+      <ol>
+        <li>A Technical Project is defined as being an open source project that meets the following criteria.
+          <ol>
+            <li>It must meet the Trademarks requirement in Section 8.</li>
+            <li>It must accept the eBPF Foundation as its primary governance structure. For example,
```

Comment:
This sounds like "the eBPF Foundation has the final say" to me. Is that correct? Also, is there a process for a Technical Project to renounce it's status? Vice versa, can the BSC withdraw the Technical Project status?